### PR TITLE
Minimum circles to be removed in C++

### PR DESCRIPTION
**Pull request by gayatri517**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

yes

### Description

Code added in cp(competitive programming) folder.

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests
#5073

### CODE REVIEW

**Comment by gayatri517** [comment]
@HarshCasper @ankitaggarwal23 please review my PR.

**Comment by gayatri517** [comment]
@siddharth25pandey please review my PR

File: C-Plus-Plus/cp/min_circles_to_be_removed.cpp [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+    else
+    {
+        return (a.first + a.second) < (b.first + b.second);
+    }
+}
+
+int main()
+{
+    int ans = 1;
+    int n, c, r;
```
Comment (ankitaggarwal23) [inline C-Plus-Plus/cp/min_circles_to_be_removed.cpp]: Use proper naming conventions don't use single letter and also while reading inputs use cout for display proper message

File: C-Plus-Plus/cp/min_circles_to_be_removed.cpp [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+    cin >> n;
```
Comment (ankitaggarwal23) [inline C-Plus-Plus/cp/min_circles_to_be_removed.cpp:28]: Use cout for display proper message.

File: C-Plus-Plus/cp/min_circles_to_be_removed.cpp [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+    vector<pair<int, int> > v;
+
+    for (int i = 0; i < n; i++)
+    {
+        cin >> c >> r;
```
Comment (ankitaggarwal23) [inline C-Plus-Plus/cp/min_circles_to_be_removed.cpp]: Use cout for display proper message.

File: C-Plus-Plus/cp/min_circles_to_be_removed.cpp [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+    for (int i = 0; i < n - 1; i++)
+    {
+        if (val <= v[i + 1].first - v[i + 1].second)
+        {
+            ans++;
+            val = v[i + 1].first + v[i + 1].second;
+        }
+    }
+
+    cout << (n - ans) <<endl;
```
Comment (ankitaggarwal23) [inline C-Plus-Plus/cp/min_circles_to_be_removed.cpp]: Use cout for display proper message.

File: C-Plus-Plus/cp/min_circles_to_be_removed.cpp [hunk 5]
```diff
@@ -0,0 +1,70 @@
...
+Output:
+2
+2)Input:
+4
+1 1 
+4 1
+5 2
+7 1
+Output:
+1
```
Comment (ankitaggarwal23) [inline C-Plus-Plus/cp/min_circles_to_be_removed.cpp]: Remove extra test case one is enough

File: C-Plus-Plus/cp/min_circles_to_be_removed.cpp [hunk 6]
```diff
@@ -0,0 +1,70 @@
...
+2)Input:
+4
+1 1 
+4 1
+5 2
+7 1
+Output:
+1 
+Time Complexity: O(nlogn)
+Space Complexity: O(nlogn) */
```
Comment (ankitaggarwal23) [inline C-Plus-Plus/cp/min_circles_to_be_removed.cpp]: Add a new line at the end

**Review by ankitaggarwal23 — CHANGES_REQUESTED** [review]

**Comment by gayatri517** [comment]
@ankitaggarwal23 I have made the changes that were requested. Please review it

File: C-Plus-Plus/cp/min_circles_to_be_removed.cpp [hunk 7]
```diff
@@ -0,0 +1,66 @@
...
+}
+/*Example:
+1)Input:
+4
+1 1
+2 1
+3 1
+4 1
+Output:
+2
```
Comment (ankitaggarwal23) [inline C-Plus-Plus/cp/min_circles_to_be_removed.cpp]: Write in a way your code is producing outputs

**Review by ankitaggarwal23 — CHANGES_REQUESTED** [review]

**Comment by gayatri517** [comment]
@ankitaggarwal23 I have made the changes. Please review

**Review by ankitaggarwal23 — APPROVED** [review]

**Pull request merged**